CMSgov/mint-app
# [EASI-2217] Read-only Model Basics - FE

# EASI-2217

## Changes and Description

- Creates new Model Basics Read Only component 
- Creates new reusable  `ReadOnlySection` component to be used throughout Read Only pages
- Creates new query and type for this page `src/queries/ReadOnly/GetAllBasics.ts`
- Adds `translateModelType` to the giant `utils/modelPlan.ts` aka translation file
- Adds a few new copy updates to `i18n` files
- Separated `TaskListStatusTag` inside of `TaskListItem` file so I can use the tag on this page too

## Acceptance Criteria
- Render questions and answers from the task list section
- Should still render unanswered questions - populated with 'No answer entered'
- Testing



## How to test this change

1. Navigate to the Model Summary ( Models on nav > Complete Plan > View this Model Plan on sidebar)
2. Ensure data is displayed correctly and matches figma


## PR Author Review Checklist



## PR Reviewer Guidelines
- It's best to pull the branch locally and test it, rather than just looking at the code online!
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
- Always make comments, even if it's minor, or if you don't understand why something was done.
